cfy_manager/components/nginx/config/rest-location.cloudify

```diff
@@ -1,4 +1,4 @@
-location ~ ^/api/v(1|2|2\.1|3|3\.1)/(blueprints|executions|deployments|nodes|events|search|status|provider|node-instances|version|evaluate|deployment-modifications|tokens|plugins|plugins-updates|snapshots|maintenance|deployment-updates|tenants|user-groups|user|users|cluster|file-server-auth|ldap|secrets|ssl|config|agents|summary|user-tokens|operations|tasks_graphs|managers|brokers|license|sites|snapshot-status|cluster-status|db-nodes)(/.*)?$ {
+location ~ ^/api/v(1|2|2\.1|3|3\.1)/(blueprints|executions|deployments|nodes|events|search|status|provider|node-instances|version|evaluate|deployment-modifications|tokens|plugins|plugins-updates|snapshots|maintenance|deployment-updates|tenants|user-groups|user|users|cluster|file-server-auth|ldap|secrets|ssl|config|agents|summary|user-tokens|operations|tasks_graphs|managers|brokers|license|sites|snapshot-status|cluster-status|db-nodes|labels)(/.*)?$ {
```

Comment:
I wonder if we should look at whether we can split this over multiple lines for readability?